package.json

```diff
@@ -97,6 +97,7 @@
     "testTimeout": 60000,
     "transformIgnorePatterns": [
       "/node_modules/(?!(@react-native|react-native)/).*/"
-    ]
+    ],
+    "clearMocks": true
```

Comment:
Is this change related to his PR? Why is it needed?

Reply:
It's not really related to this pr but I noticed that mocks weren't automatically cleared because I had to add tests on debug and since the calls to console.log were not cleared, my tests did not pass after I ran them with the other ones. I think tests should all pass when they are run by themselves and should not depend on other tests and automatically clearing mocks call history helps achieving that